### PR TITLE
A Very Big Sum

### PR DESCRIPTION
**Pull request by lina2000**
A Very Big Sum on https://www.hackerrank.com/challenges/a-very-big-sum

### CODE REVIEW

File: A Very Big Sum/source.c [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
```
Comment (kt-9) [inline A Very Big Sum/source.c]: Please from now on, include the link to the problem on the first line of that this code is solving in comments like so:
```
/* Solution to https://www.hackerrank.com/contests/d1/challenges/a-very-big-sum */
```

File: A Very Big Sum/source.c [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+int main() {
+    long n = 0; 
+    scanf("%lu", &n);
```
Comment (kt-9) [inline A Very Big Sum/source.c]: `%lu` is used for `long unsigned int`. You can assume itcan hold values in range [0, 2^64).

It works in this problem, but you should also know the other formatting options like one for `signed long int` which can hold values in range [2^63, 2^63).

File: A Very Big Sum/source.c [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
+
+int main() {
+    long n = 0;
```
Comment (kt-9) [inline A Very Big Sum/source.c]: `n` is the number of integers, which is up to 10 for this problem. You don't need to store it in `long`.

File: A Very Big Sum/source.c [hunk 4]
```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
+
+int main() {
+    long n = 0; 
+    scanf("%lu", &n);
+    long x = 0;
```
Comment (kt-9) [inline A Very Big Sum/source.c]: `x` is the number of integers, which is up to 10 for this problem. You don't need to store it in `long`.

File: A Very Big Sum/source.c (hunk 1 above)
Comment (kt-9) [reply A Very Big Sum/source.c]: Also put your file `source.c` into the folder called `a-very-big-sum`, not A Very Big Sum

**Comment by lina2000** [comment]
Answer my messege, please.
Should I add /* Solution to
https://www.hackerrank.com/challenges/compare-the-triplets */   at the D1/
compare-the-triplets/source.c code ?

2017-10-16 19:55 GMT+06:00 Nurlan Kanapin <notifications@github.com>:

File: a-very-big-sum [hunk 5]
```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>   /* Solution to https://www.hackerrank.com/contests/d1/challenges/a-very-big-sum */
```
Comment (kt-9) [inline a-very-big-sum]: Put it on a separate line before `#include ...`

File: a-very-big-sum [hunk 6]
```diff
@@ -0,0 +1,17 @@
+/* Solution to https://www.hackerrank.com/contests/d1/challenges/a-very-big-sum */
```
Comment (kt-9) [inline a-very-big-sum]: Put your file into a folder a-very-big-sum/source.c

**Pull request merged**